src/wallet/RelayTransactionResponse.ts

```diff
@@ -54,17 +63,14 @@ export class RelayTransactionResponse implements TransactionResponse {
     return this.safeTxHash
   }
 
-  public async wait(): Promise<any> {
+  public async wait(): Promise<TransactionReceipt> {
```

Comment:
With the changes made to the wait and the sendTransaction return params, we probably should update the README to reflect the modification (and the doc as well)